pyvista/plotting/affine_widget.py

```diff
@@ -0,0 +1,372 @@
+"""Affine widget module."""
+import numpy as np
+
+import pyvista as pv
+from pyvista.core.errors import VTKVersionError
+from pyvista.core.utilities.misc import try_callback
+
+from . import _vtk
+
+DARK_YELLOW = (0.9647058823529412, 0.7450980392156863, 0)
+INDEX_MAPPER = {0: 1, 1: 2, 2: 0}
+AXES = {
+    0: np.array([1, 0, 0], dtype=float),
+    1: np.array([0, 1, 0], dtype=float),
+    2: np.array([0, 0, 1], dtype=float),
+}
+
+
+def _make_quarter_arc():
+    """Make a quarter circle centered at the origin."""
+    circ = pv.Circle(resolution=100)
+    circ.faces = np.empty(0, dtype=int)
+    circ.lines = np.hstack(([26], np.arange(0, 26)))
+    return circ
+
+
+def get_angle(v1, v2):
+    """Compute the angle between two vectors in degrees.
+
+    Parameters
+    ----------
+    v1 : numpy.ndarray
+        First input vector.
+    v2 : numpy.ndarray
+        Second input vector.
+
+    Returns
+    -------
+    float
+        Angle between vectors in degrees.
+    """
+    theta = (np.arccos(np.clip(np.dot(v1, v2), -1.0, 1.0))) * (180 / np.pi)
```

Comment:
Let's use the function to clarify the intent of your code.
```suggestion
    theta = np.rad2deg(np.arccos(np.clip(np.dot(v1, v2), -1.0, 1.0)))
```